site/app/templates/admin/Docker.twig

```diff
@@ -5,79 +5,141 @@
+                {% endif %}
+            {% endfor %}
+            <td>{{worker['num_autograding_workers']}}</td>
+        </tr>
+        {% endfor %}
+    </table>
 </div>
 
 <div class="content">
-    <h2 class="docker-header">System Wide Docker Images </h2>
-    <button class="btn btn-default"
-            onclick="collapseSection('system-images','system-images-collapse')"
-            id="system-images-collapse">
-            Collapse
-    </button>
-
-    <div id="system-images" style="display:block;">
-        <p class="black-message">These are the docker images that are installed on the primary submitty machine</p> 
-        <div class="docker-images-container">
-            {{ _self.renderDockerImage(autograding_containers['all_images']) }}
+    <h2 class ="docker-header">Autograding Docker Images</h2>
+    {% if capabilities is not null %}
+        <div class="filter">
+            <div style="display:inline-block;">Capability Filter: </div>
+            <div style="display:inline-block;">
+                <button class="btn btn-primary" id="show-all">Show All</button>
+                {% for capability in capabilities %}
+                    <button data-capability="{{capability}}" class="btn filter-buttons" style="background-color:{{capability_to_color[capability]}}; border-color: {{capability_to_color[capability]}}">
+                        {{capability}}
+                    </button>
+                {% endfor %}
+            </div>
         </div>
-    </div>
+    {% endif %}
+    
+    <table class="table striped-table docker-table mobile-table">
+        <thead>
+            <tr>
+                <th>Image Name</th>
+                <th>Tag</th>
+                <th>ID</th>
+                <th>Alias</th>
+                <th>Created Date</th>
+                <th>Capabilities Containing This Image</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for key, value in autograding_containers['all_images'] %}
+            <tr class="image-row" data-error={{value["name"] in fail_images ? "true" : "false"}}>
+                <td>{{ value["name"] }}</th>
+                <td>{{ value["tag"] }} </td>
+                <td>{{ value["short_id"] | trim("sha256:") }} </td>
+                <td>
+                {% if value["additional_names"] | length > 0 %}
+                <ul id="tag-list">
+                    {% for name in value["additional_names"] %}
+                    <li>{{ name }}</li>
+                    {% endfor %}
+                </ul>
+                {% endif %}
+                </td>
+                <td> {{ value["created"] }} </td>
+                <td>
+                    {% if value["name"] ~ ':' ~ value["tag"] in image_to_capability|keys %}
+                        {% for capability in image_to_capability[value["name"] ~':'~ value["tag"]] %}
+                            <span class="badge" style="background-color:{{capability_to_color[capability]}}">{{capability}}</span>
```

Comment:
The background colors with white font is causing WAVE contrast issues, the background should be darker. The ones I saw have issues are:

- #c3a2d2 (purple)
- #99B270 (yello-green)
- #6bb88f (green)
- #C8938D (brown)
- #cd98aa (pink)

Reply:
Because the colors used for the badges are fixed for both light and dark mode, I had to change the table to be fixed in color scheme as well. This means I won't be using the variables specified in `colors.css`, is this an acceptable solution?

Reply:
Yeah I think thats fine, these badges seem unique to this page so if we need to reuse them in the future we work on getting them back into colors.css but for now it should be fine

Reply:
Actually... while the badges are unique to this page, the colors are copied from office hours queue, would that be cause for a small refactor?

Reply:
I would say thats up to you, since you're generating the list of colors from php the most you could use the css `var(--your-color)` syntax and then move the hex codes to colors.css

Since it sounds like that touches office hours I think it should be in its own PR or you can make an issue for it